assets/app/scripts/controllers/create.js

```diff
@@ -14,8 +14,6 @@ angular.module('openshiftConsole')
 
     $scope.templatesByTag = {};
 
-    $scope.sourceURLPattern = /^((ftp|http|https|git):\/\/(\w+:{0,1}\w*@)|git@)?([^\s@]+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/;
```

Comment:
keep the pattern in the controller... it's used by `assets/test/spec/controllers/create.js`